src/components/features/Poper.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import Box from '@mui/material/Box'
+import Popper, { PopperPlacementType } from '@mui/material/Popper'
+import Typography from '@mui/material/Typography'
+import Grid from '@mui/material/Grid'
+import Button from '@mui/material/Button'
+import Fade from '@mui/material/Fade'
+import Paper from '@mui/material/Paper'
+import InfoIcon from '@mui/icons-material/Info'
+import { useTranslation } from 'react-i18next'
+
+interface GoodType {
+  title: string
+  value: string
+  quantity: number
+}
+
+interface PositionedPopperProps {
+  goodTypes: GoodType[]
+}
+
+export default function PositionedPopper({ goodTypes }: PositionedPopperProps) {
```

Comment:
make a better name, Popper says nothing what this is about.
You can also make it reusable.
Put the Popper into its own and then for the tooltip you just use it.

Reply:
Fixed